content/en/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,390 @@
+---
+reviewers:
+- sftim
+title: LimitRange
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on kubernetes cluster.
+With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
+Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.
+there is a concern that one Pod/Container could monopolise its namespace resources.

```

Comment:
Delete these last two lines and replace with:

LimitRange is a tool that provides constraints to prevent a Pod from monopolizing its namespace resources.